program_info/CMakeLists.txt

```diff
@@ -8,44 +8,44 @@ if(UNIX)
     endif()
 endif()
 
-set(Launcher_CommonName "PolyMC")
+set(Launcher_CommonName "PrismLauncher")
 
-set(Launcher_Copyright "PolyMC Contributors\\n© 2012-2021 MultiMC Contributors")
+set(Launcher_Copyright "Prism Launcher Contributors\\n© 2012-2021 MultiMC Contributors")
```

Comment:
Don't we have to mention PolyMC Contributors anyway, and add Prism Launcher Contributors to it instead?

Reply:
Prism Launcher contributors = PolyMC contributors

Reply:
Because the license is the same?

Reply:
I would also agree the true PolyMC contributors are all moving to Prism Launcher

Reply:
> Prism Launcher contributors = PolyMC contributors

Can we get a legal-ish opinion in here on what to actually do?

Reply:
Legally, it doesn't matter. The copyright headers in the code matter. This is just a text that is shown in some places

Reply:
By that logic the MultiMC contributors shouldn't need to be mentioned either right?

Reply:
Yes. But I am fine with MultiMC people